test/jdk/java/awt/PopupMenu/PopupMenuLocation.java

```diff
@@ -106,22 +115,33 @@ private void show(MouseEvent e) {
     }
 
     private static void openPopup(final Frame frame) throws Exception {
-        Robot robot = new Robot();
-        robot.setAutoDelay(200);
         robot.waitForIdle();
-        Point pt = frame.getLocationOnScreen();
-        robot.mouseMove(pt.x + frame.getWidth() / 2, pt.y + 50);
+        Point loc = frame.getLocationOnScreen();
```

Comment:
Since you removed the reading the location from EDT via a separate loc variable, you do not need to rename pt here? The only meaningful change is adding the CountDownLatch?

Reply:
Reverted it